Sources/MessagingInApp/Views/InAppMessageView.swift

```diff
@@ -127,6 +135,7 @@ public class InAppMessageView: UIView {
             // We are already displaying or rendering a messsage. Do not show another message until the current message is closed.
             // The main reason for this is when a message is tracked as "opened", the Gist backend will not return this message on the next fetch call.
             // We want to coninue showing a message even if the fetch no longer returns the message and the message is currently visible.
+            refreshViewListener?()
```

Comment:
Wondering if `refreshViewListener` is required to be call in all conditions then we could put it in the start of the method or defer ? Or there's a reason that it's been added this way ?